templates/submission/review-files.tpl

```diff
@@ -0,0 +1,55 @@
+{**
+ * templates/submission/review-files-step.tpl
+ *
+ * Copyright (c) 2014-2022 Simon Fraser University
+ * Copyright (c) 2003-2022 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * The template in the submission wizard when reviewing the upload files step.
+ *}
+<div class="submissionWizard__reviewPanel">
+    <div class="submissionWizard__reviewPanel__header">
+        <h3 id="review{$step.id}">
```

Comment:
Thanks for the reminder. I think I've got all the occurrences on the submission wizard templates.

Reply:
I think this might've gone too far -- I see one case where HTML is now being escaped:
![image](https://user-images.githubusercontent.com/200411/198075933-5e9da2a2-c397-4bc7-ad1d-526b1c04613d.png)

Reply:
Woops! Fixed that. :)